packages/templates/usa-sign-in/usa-sign-in-form/usa-sign-in-form.twig

```diff
@@ -16,18 +18,25 @@
     >
     {# /Email #}
     {# Password #}
-    <label class="usa-label" for="{{ password.input.id |default('password-sign-in') }}">
+    <label class="usa-label" for="{{ password_id }}">
       {{ password.label |default('Password')}}
     </label>
     <input
       class="usa-input"
-      id="{{ password.input.id |default('password-sign-in')}}"
+      id="{{ password_id }}"
       name="password"
       type="password"
       required
     >
     {% include "@components/_usa-password/src/usa-password.twig" with {
-      toggle: password.toggle
+      "toggle": {
+        "title": "Toggle password",
+        "aria_controls": password_id,
+        "text": {
+          "show": "Show password",
+          "hide": "Hide password"
+        }
+      }
```

Comment:
Discovered this quirk while testing the [sign-in-form component on Site](https://github.com/uswds/uswds-site/pull/1493/files#r850718034).

**What didn't work**
Data isn't passed when building to HTML by reference in twig includes.

```twig
{% include "@components/_usa-password/src/usa-password.twig" with { toggle: password.toggle } %}
```

**What worked**
You can still use includes, but you have to **write** out the data you want to pass.

```js
{% include "@components/_usa-password/src/usa-password.twig" with { 
  toggle: {
    …the data
  }
} %}
```

Reply:
Thanks for the digging into that @mejiaj. Much appreciated!